compiler/src/iree/compiler/Codegen/Common/test/lower_ukernel_to_calls.mlir

```diff
@@ -81,7 +81,88 @@ func.func @generic_ukernel(%arg0 : memref<?x?xf32>, %arg1 : memref<?x?xf32>,
 
 // -----
 
-func.func @mmt4d_ukernel(%arg0 : memref<?x?x?x?xf32>, %arg1 : memref<?x?x?x?xf32>,
+func.func @ukernel_generic_raw_scalar_types(%arg0: i32, %arg1 : f64, %arg2 : index, %arg3 : memref<f32>) {
+  iree_codegen.ukernel.generic_raw "scalar_fn" ins(%arg0, %arg1, %arg2 : i32, f64, index) outs(%arg3 : memref<f32>)
+  return
+}
+//      CHECK: func.func private @scalar_fn(i32, f64, index, memref<f32>)
+//      CHECK: func.func @ukernel_generic_raw_scalar_types
+// CHECK-SAME:     %[[ARG0:[a-zA-Z0-9]+]]: i32
+// CHECK-SAME:     %[[ARG1:[a-zA-Z0-9]+]]: f64
+// CHECK-SAME:     %[[ARG2:[a-zA-Z0-9]+]]: index
+// CHECK-SAME:     %[[ARG3:[a-zA-Z0-9]+]]: memref<f32>
+//      CHECK:   %[[BASE:.+]], %[[OFFSET:.+]] = memref.extract_strided_metadata %[[ARG3]]
+//      CHECK:   call @scalar_fn(%[[ARG0]], %[[ARG1]], %[[ARG2]], %[[BASE]])
```

Comment:
I think this needs to include offset too.

Reply:
Right, this is a preexisting issue and being addressed in #13054.

Reply:
As discussed on Discord, updated this PR to add the commit from #13054.